lcls-twincat-motion/Library/POUs/Motion/PMPS/FB_PositionStatePMPS.TcPOU

```diff
@@ -92,6 +93,11 @@ bAtSafeState := F_SafeBPCompare(stHighBeamThreshold, stBeamNeeded);
 ffBeamParamsOk.i_xOK := bFFOxOk;
 ffBeamParamsOk.i_xReset S= bFFOxOk AND bAtSafeState;
 ffBeamParamsOk.i_xReset R= NOT ffBeamParamsOk.i_xOK;
+
+IF bAutoReset THEN
+    ffBeamParamsOk.i_xAutoReset := True;
+END_IF
```

Comment:
This is strangely indirect, why not something like this?
```suggestion
ffBeamParamsOk.i_xAutoReset := bAutoReset;
```

Reply:
right, this an unnecessary if statement.